src/recipe/recipe.controller.ts

```diff
@@ -96,12 +110,14 @@ export class RecipeController {
     @Param('id', ParseIntPipe) id: number,
     @Body() updateRecipeRequest: UpdateRecipeRequest,
   ): Promise<UpdatedRecipeResponse> {
-    const updatedRecipe = await this.recipeService.updateRecipe(
-      id,
-      updateRecipeRequest,
-    );
+    let recipe: Recipe = await this.redisCacheService.get(`${id}`);
+
+    if (!recipe) {
+      recipe = await this.recipeService.updateRecipe(id, updateRecipeRequest);
+      await this.redisCacheService.del(`cache:recipe:${recipe.id}`);
+    }
```

Comment:
not sure what's happening here - why do we need to fetch a recipe from cache to update it?